src/lib/rmt/wizard_final_page.rb

```diff
@@ -67,10 +85,38 @@ def back_handler
   end
 
   def run
-    unless (Yast::Service.Enable('rmt-server') && Yast::Service.Restart('rmt-server'))
-      Yast::Report.Error(_("Failed to enable and restart service 'rmt-server'"))
-    end
+    rmt_service_start
     render_content
     event_loop
   end
+
+  def rmt_service_start
```

Comment:
This should be reorganized a bit, I feel.
As this method (`rmt_service_start`) handles the dialog, does the actual service restarting and conditionally calls another method (`rmt_enable_timers`).
Please remove this nesting. Maybe you can do all of that in in the run method? Or better split out the actual actions into methods (like already done well in `rmt_enable_timers`), but not mix concerns.

And maybe by making this clearer, you also spot why it currently fails (for me) with:
![oh_no_yast](https://user-images.githubusercontent.com/5372947/40060545-2c50bf5c-5857-11e8-96fe-d400ddd747af.png)

I get this after the _success_ message appeared.